SwiftDashboard.md

```diff
@@ -0,0 +1,56 @@
+# Firebase Swift Modernization Dashboard
+
+This dashboard summarizes the status of Firebase's [2022 Swift Modernization Project](Roadmap.md).
+Please upvote or create a [feature request](https://github.com/firebase/firebase-ios-sdk/issues)
+to help prioritize any particular cell(s).
+
+|                       | AB  | An     | ApC    | ApD    | Aut    | Cor    | Crs    | DB     | Fst    | Fn     | IAM    | Ins    | Msg    | MLM    | Prf    | RC     |    Str |
+|   :---                | :--- | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: |
+| **Swift Library**     | ❌   |   ✔   | ❌     |❌     | ❌     | ❌     | ❌      |  ✔     |  ✔    | 1      |  ✔     | ❌    | ❌     | ✔      | ❌     | ❌    | ✔     |
+| **API Tests**         | ❌   |  ❌    |  ✔    |❌     | ❌     | ✔       | ❌     | 3      | 2     |  ✔     | 2      | ❌     | ❌     | 2      | ❌    |  ✔     | ✔    |
+| **async/await**       | ❌   |  ❌    |  ✔    |❌     | ❌     |  ✔      | ❌     | 3     | ❌     | ❌     | ❌     | ❌    | ❌     | ❌    | ❌     |  ✔    | ✔    |
+| **Swift Errors**      |  ❌  |  ❌    | ❌    |❌     | 4      | ❌     | ❌     | ❌     | ❌    | ❌     | ❌     | ❌    | ❌     | ✔      | ❌     | ❌   | 5   |
+| **Codable**           | n/a  | n/a     | n/a   |n/a    | n/a     | n/a    |n/a     |  ✔     |  ✔     | 1      | n/a     | n/a   | ❌     | n/a    | n/a    | n/a   |n/a   |
+| **SwiftUI Lifecycle** | n/a  |  ❌    | n/a    |❌     | ❌     | n/a    |n/a     | n/a    | n/a    | n/a     | n/a    | n/a   | ❌     | n/a    | n/a    | n/a   |n/a  |
+| **Property Wrappers** |  ❌  |  ❌    | ❌    |❌     | ❌     | ❌     | ❌     | ❌     | ✔     | ❌     | ❌     | ❌    | ❌     | ❌    | ❌     | ❌   |❌     |
```

Comment:
Thanks for putting this together! I'm torn about having a specific row for Property Wrappers since it's an implementation detail and may not (likely doesn't?) make sense for all products. For example, Functions may not make sense to have any property wrappers but feels wrong to have an X or even N/A there.

I sort of think of it in the same way as a row for enums with associated types, we'd love to add them where the API makes sense but shouldn't add them just because we can.

Reply:
I think that property wrappers might be similar to Codable. We should think about opportunities to add them and then go to either n/a or ❌  /✔

Reply:
Sounds reasonable to me, thanks.